### DISCUSSION TITLE
Would you contribute if the MTG Deck Analyzer were open sourced?

### DISCUSSION THREAD

**Discussion by svidgen** [svidgen/mtg.deckanalyzer.com-Feedback#45] [Polls]
My code is ugly. The site is actually very very beta under the hood. And, I haven't had much time to work on it lately, other than to tinker a little with what I'd need to do to migrate to AWS. (Thinking of AWS Amplify)

I would be looking for help with features, fixes, *and possibly* help migrating the site to AWS.

**Comment by Maleware** [comment]
Depends on the language you wrote this thing ;) My analyzer is build in Rust, so I would prefer a relatable language
